examples/secp256k1-ecdsa-verify/guest/src/lib.rs

```diff
@@ -0,0 +1,16 @@
+#![cfg_attr(feature = "guest", no_std)]
+
+use jolt_inlines_secp256k1::{ecdsa_verify, Secp256k1Fr, Secp256k1Point};
+
+// verifies an secp256k1 ECDSA signature
+// given message hash z, signature (r, s), and public key Q
+// all inputs are little-endian u64 arrays in normal form
+// the function panics (and spoils the proof) if the signature is invalid
```

Comment:
I think the proof should still succeed and be verifiable if signature is invalid. Function should just panic.
Now, the proof should be spoiled if untrusted advice is malicious.